.circleci/config.yml

```diff
@@ -117,7 +117,7 @@ workflows:
             branches:
               only: master
 
-      - site-deploy: # This deploys to our stage site at https://stage.rally-web.nonprod.dataops.mozgcp.net
+      - site-deploy: # This deploys to our stage site at https://stage.rally-web.nonprod.dataops.mozgcp.net / https://members.rally.allizom.org
```

Comment:
I assume this intentional obfuscation. I'd not seen this particular domain before.

Reply:
allizom.org? its used by some teams (and not others) for mozilla stage (you probably already noticed, its just mozilla spelled backwards). I find it easier to remember myself than the env.name.realm.team.mozgcp.net guy (for whatever reason).

Reply:
(i will happily remove this though if you prefer not to have it)

Reply:
nah. keep it.